metadata-models/src/main/pegasus/com/linkedin/metadata/recommendation/RecommendationRenderType.pdl

```diff
@@ -28,5 +28,10 @@ enum RecommendationRenderType {
   /**
    * A list of recommended searches
    */
-  SEARCH_QUERY_LIST
+  SEARCH_QUERY_LIST,
+
+  /**
+   * List of domains to search
+   */
+  DOMAIN_SEARCH_LIST
```

Comment:
do we hide this in the case there are no domains? Or have a prompt to create a domain?

Reply:
yup we hide it! in case of 0 domains